crates/script/src/lib.rs

```diff
@@ -211,6 +211,10 @@ impl ScriptArgs {
 
         let (config, mut evm_opts) = self.load_config_and_evm_opts_emit_warnings()?;
 
+        // print warning message if `cbor_metadata` is disabled
+        let msg = "Warning! \"--no-metadata\" flag or \"cbor_metadata\" set to false can result in deployment failures.".yellow();
+        p_println!(!config.cbor_metadata => "{msg}");
```

Comment:
should use cli_warn

Reply:
:+1: noted, thank you (I used the coverage warn style, but will update that one to cli warn as well), reworking the whole PR a little bit so changed to draft